mu-plugins/central-hub/tests/phpunit/unit/meta-data/saveMetaBoxes.php

```diff
@@ -0,0 +1,155 @@
+			->once()
+			->with( 'meta_box.members' )
+			->andReturn( 'members' );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\is_okay_to_save_meta_box' )
+			->once()
+			->with( 'members' )
+			->andReturn( false );
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getConfigParameter' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\save_custom_fields' )->never();
+		save_meta_boxes( 317 );
+	}
+
+	/**
+	 *  Test save_meta_boxes() should not save when the nonce key is not equal to the expected value.
+	 */
+	public function test_should_not_save_when_the_nonce_key_is_not_equal_to_the_expected_value() {
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_keys' )
+			->once()
+			->withNoArgs()
+			->andReturn( [ 'meta_box.reviews' ] );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_id' )
+			->once()
+			->with( 'meta_box.reviews' )
+			->andReturn( 'reviews' );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\is_okay_to_save_meta_box' )
+			->once()
+			->with( 'reviews' )
+			->andReturn( false );
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getConfigParameter' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\save_custom_fields' )->never();
+		save_meta_boxes( 342 );
+	}
+
+	/**
+	 * Test save_meta_boxes() should not save when the expected nonce fails.
+	 */
+	public function test_should_not_save_when_the_expected_nonce_fails() {
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_keys' )
+			->once()
+			->withNoArgs()
+			->andReturn( [ 'meta_box.members' ] );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_id' )
+			->once()
+			->with( 'meta_box.members' )
+			->andReturn( 'members' );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\is_okay_to_save_meta_box' )
+			->once()
+			->with( 'members' )
+			->andReturn( false );
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getConfigParameter' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\save_custom_fields' )->never();
+		save_meta_boxes( 349 );
+	}
+
+	/**
+	 * Test save_meta_boxes() should save when one or more valid post meta values are added.
+	 */
+	public function test_should_save_when_one_or_more_valid_post_meta_values_are_added() {
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_keys' )
+			->once()
+			->withNoArgs()
+			->andReturn( [ 'meta_box.members' ] );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_id' )
+			->once()
+			->with( 'meta_box.members' )
+			->andReturn( 'members' );
+		Monkey\Functions\when( 'spiralWebDB\Metadata\is_okay_to_save_meta_box' )
+			->justReturn( 'wp_verify_nonce' );
+		Monkey\Functions\when( 'KnowTheCode\ConfigStore\getConfigParameter' )
+			->justReturn( [
+				'role'            => 'Soprano',
+				'residence_city'  => 'Chicago',
+				'residence_state' => 'IL',
+				'tour_number'     => '3',
+			] );
+		Monkey\Functions\when( 'spiralWebDB\Metadata\save_custom_fields' )
```

Comment:
@rgadon107 We expect this mock to run once, with the 3 arguments, and return `null`.